### PR TITLE
JBPM-5999 - Turn checkstyle violations on in designer and reformat ex…

### PR DESCRIPTION
**Pull request by tsurdilo**
…iting java code

### CODE REVIEW

**Comment by tsurdilo** [comment]
jenkins please retest this

**Comment by tsurdilo** [comment]
build fail not related to this pr (or designer). will rebuild when all is stable

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by hasys** [comment]
Jenkins retest please

File: jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonMarshaller.java [hunk 1]
```diff
@@ -119,1016 +234,1593 @@ public String marshall(Definitions def, String preProcessingData) throws IOExcep
...
-        generator.writeEndArray();
-        generator.writeArrayFieldStart("outgoing");
-        generator.writeEndArray();
-
-        generator.writeEndObject();
-    } **/
-
-    protected void marshallCallableElement(CallableElement callableElement, Definitions def, JsonGenerator generator) throws JsonGenerationException, IOException {
+    /**
+     * protected void marshallMessage(Message message, Definitions def, JsonGenerator generator) throws JsonGenerationException, IOException {
```
Comment (jomarko) [inline jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonMarshaller.java]: @tsurdilo probably worth to delete completely this `marshallMessage` if it is not used.

**Comment by tsurdilo** [comment]
@jomarko good point! amended

**Comment by hasys** [comment]
Jenkins retest please

**Pull request merged**